int16 gripperset
float64 JointState  
string putdown_state11